docs/console/company-configuration/extensions.md

```diff
@@ -4,75 +4,90 @@ title: Manage Extensions
+Extensions can be managed through a dedicated section called **Extensibility** placed in the Company Overview page.
 
-![extension placeholder](./img/extension-placeholder.png)
+:::info
+Only **Company Owners** have the necessary permissions to see and use this section.
+Moreover, extensions are **only accessible within the Company** they are registered into.
+:::
 
-## Create Extension
+![extension placeholder](./img/extensions/extension-placeholder.png)
 
-When accessing this section for the first time or from the extensions table, you can create new extensions by clicking the `Create Extension` button. This will open a modal with the following fields to fill:
+There are different extension types supported:
 
-- `Extension name` (_required_): Choose a name for your extension
-- `Entry URL` (_required_): Enter the URL of the web page that will load when accessing your extension menu item and be embedded inside an iframe
-- `Destination Area` (_required_): Select the destination where your extension menu item will be placed
-- `Category` (_required_): Select one of the menu groups that will contain your extension menu item
-- `Visibility`: Check the contexts in which you want to make your extension visible. If no boxes are checked, the extension will be inactive.
+- iframe: allows you to embed web pages in Console using an `<iframe>` tag
+- External link: these extensions will be shown as menu items in the sidebar but opened in a new window on click
 
-:::info
-The `Category` options are based on the selected destination area.
+:::tip
+Embedding a web app as an *iframe* extension may require additional configurations,
+such as making sure `x-frame-options` header or the `frame-ancestors` Content Security Policy are properly configured
 :::
 
+## Add new Extension
+
+In any moment, you can add a new extensions to your Company; creating an extension requires the following configurations to be filled:
+
+- **Extension name** (*required*): the name for your extension
+- **Entry URL** (*required*): the URL of the web page that will load when accessing your extension (This will be embedded inside an iframe or opened in a new window depending on the extension type)
+- **Destination Area** (*required*): the destination where your extension menu item will be placed
+- **Category** (*required*): the menu category that will contain the extension menu item
+- **Visibility**: contexts in which you want to make your extension visible. If nothing is checked, the extension will be inactive.
+
 :::info
-The `Visibility` can be applied only on the `Whole Company` context when the `Company Overview` destination area is selected.
+The **Category** options are based on the selected destination area.
+
+The **Visibility** can be applied only on the *Whole Company* context when the *Company Overview* destination area is selected.
```

Comment:
I would put the info related to each field close to where the field is described to avoid confusion

```suggestion
When the *Company Overview* destination area is selected, the only available **Visibility** option will be the *Whole Company*.
```